object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -31,6 +34,11 @@
  */
 public class ObjectConstructionAnnotatedTypeFactory extends CalledMethodsAnnotatedTypeFactory {
 
+  /**
+   * Bidirectional map to preserve temporal variables created for nodes with non-empty @MustCall
+   * annotation and the corresponding nodes
+   */
+  protected BiMap<LocalVariableNode, Tree> mapTempVarToNode = HashBiMap.create();
```

Comment:
Just `tempVarToNode`:
```suggestion
  protected BiMap<LocalVariableNode, Tree> tempVarToNode = HashBiMap.create();
```

Reply:
https://github.com/kelloggm/object-construction-checker/pull/291/commits/790ac07e99314570d32d4c8b2fc03158226ec15a